udata/harvest/tasks.py

```diff
@@ -19,15 +19,18 @@ def harvest(self, ident):
     Backend = backends.get(current_app, source.backend)
     backend = Backend(source)
     items = backend.perform_initialization()
-    if items > 0:
+    if items is None:
```

Comment:
Thank you! This will fix some current errors raised in Sentry: https://errors.data.gouv.fr/organizations/sentry/issues/199/